### ISSUE TITLE
[1.20.1] Game crashes when generating Soul Shards in multiplayer

### ISSUE THREAD

**Issue by Visual-Anxiety** [Alexthw46/Eidolon-Repraised#38]
Using the same modpack and forge version as my previous issue (the one about the raven cloak causing a crash in multiplayer), soul shards cause a crash when generated either with the crystallization ritual or with the reaper's scythe. When loading back into the game, the crystals are created successfully.

![image](https://github.com/Alexthw46/Eidolon-Repraised/assets/155265708/670d4f4a-7fc0-4976-a9d8-528095b3acdc)

**Comment by Alexthw46** [comment]
i rechecked if it was a server only thing but it works on my end on server, we also have a test server and it didn't trigger anything weird. (same applies for raven cloak) i can only think of a mod messing up with packets but i have no idea why

**Comment by Alexthw46** [comment]
ok, it must be tied to the last update. managed to trigger something on said test server, but only a freeze for the cloak

**Comment by Visual-Anxiety** [comment]
I updated the raven cloak issue after more testing. Apparently, it only causes this issue when the bar for the raven cloak is completely full, and only on multiplayer. Was this similar to what you encountered on the test server?

**Comment by Alexthw46** [comment]
Nah, it seems that the issue i had was just a weird thing, no actual freeze, disconnect or crash but my screen stopped refreshing the rendering, i could theorically blind-play. And the soul sharding was fine too.

I tried again and the raven cloak was fine, did you make sure the client and the server have the same version of eidolon? Usually it doesn't allow connecting when there are desync issues, but you never know...
This is totally tied to packets but the only thing that changed from the prev version about them is adding a new one (which shouldn't be currently in use)
